test/functional/abandonconflict.py

```diff
@@ -73,8 +73,8 @@ def run_test(self):
 
         # Restart the node with a higher min relay fee so the parent tx is no longer in mempool
         # TODO: redo with eviction
-        stop_node(self.nodes[0],0)
-        self.nodes[0]=start_node(0, self.options.tmpdir, ["-minrelaytxfee=0.0001"])
+        self.stop_node(0)
+        self.nodes[0] = self.start_node(0, self.options.tmpdir, ["-minrelaytxfee=0.0001"])
```

Comment:
The plan is to change `start_node` so assignment here would be unnecessary in the future?